lib/tbot/identity/identity.go

```diff
@@ -240,6 +241,10 @@ func (i *Identity) SSHClientConfig() (*ssh.ClientConfig, error) {
 		return nil, trace.BadParameter("user cert has no valid principals")
 	}
 	return &ssh.ClientConfig{
+		// TODO: Should we be limiting KEX, MAC and Ciphers here to
```

Comment:
Outstanding TODO. Existing codebase does not configure the FIPS compliant KEX algos, MAC algos and Ciphers for created SSH clients, only for servers. I'm unsure if there's a requirement that we configure these algos for clients.

Reply:
The answer is: yes, clients should also restrict their algo sets.